src/firefly/js/ui/DropDownMenu.jsx

```diff
@@ -4,7 +4,7 @@ import {isFunction} from 'lodash';
 import React, {useContext, useEffect, useRef, useState} from 'react';
 import {Box, Button, Card, Stack, Tooltip} from '@mui/joy';
 import PropTypes from 'prop-types';
-import {DropDownDirCTX} from './DropDownDirContext.js';
+import {DropDownDirCtx} from './DropDownDirContext.js';
```

Comment:
Are you aware that JoyUI has a Dropdown component that will handle all of these things for you?  I've converted table's dropdown to it.

Reply:
In a PR a few weeks ago I documented how the JoyUI dropdown would be very difficult to convert to. We would have to rewrite a lot of code. I experimented with it a lot. Also I don't think is does cascades, but I never got that far.